### PR TITLE
Setup bors

### LINKED ISSUE #4: Setup bors.toml
- Setup `bors.toml` for auto-merging

### PR DESCRIPTION
**Pull request by utilForever**
This revision includes;
- Setup bors (Resolves #4)
  - Add bors.toml

### CODE REVIEW

**Review by sboh1214 — DISMISSED** [review]

File: bors.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+status = [
+    "Server Audit"
+    "Server Build"
+    "Server Clippy"
+    "Server Coverage"
+    "Server Fmt"
+    "Server Test"
+]
+delete_merged_branches = true
+timeout_sec = 1200 # 20 min
```
Comment (buttercrab) [inline bors.toml:10]: Maybe a empty line at the end?
```suggestion
timeout_sec = 1200 # 20 min

```

**Review by buttercrab — APPROVED** [review]

**Review by buttercrab — DISMISSED** [review]

**Review by sboh1214 — DISMISSED** [review]

**Review by buttercrab — APPROVED** [review]

**Review by sboh1214 — APPROVED** [review]

**Pull request closed without merging**